README.md

```diff
@@ -363,7 +363,7 @@ and if they sent the correct amount, they get the Moment back.
 There are two versions of the Top Shot Market Contract.
```

Comment:
Should we make reference to there being the intermediary `TopShotMarketV2` version - and why V3 was created on top of it?

Reply:
yeah I can add a line